### ISSUE TITLE
New feature to BatchGenerator: inverse_transform

### ISSUE THREAD

**Issue by maxsch3** [maxsch3/keras-batchflow#18]
a model that uses BatchGenerator returns predictions in the format of y structure. The individual outputs it is consists of are encoded by the sklearn feature encoders and they need to be decoded individually.

A convenience method is required that will automate decoding and will collect the output into pandas dataframe

**Comment by maxsch3** [comment]
Done and merged

**Comment by maxsch3** [comment]
This hasn't been done well:
1. The inverse_transform was not added to BatchGenerator. 
2. inverse_transform does not account for a fact that keras model usually returns output in one-hot encoded format even though labels may have been provided as integers.

**Comment by maxsch3** [comment]
For the latter, I think it should be a developer's responsibility, who should use one-hot encoder to encode labels. This way, it will work as is.

There is no clear reason to complicate the module by adding the automatic conversion from one-hot results to integer in case LabelEncoder and sparse_categorical crossentropy were used in keras

**Comment by maxsch3** [comment]
re-opening
